frontend/src/app/pages/auth-signin/auth-signin.component.ts

```diff
@@ -90,8 +90,6 @@ export class AuthSigninComponent implements OnInit {
   }
 
   idpConnectHello($event){
-
     this.authService.IdpConnect('hello')
-      .then(console.log)
```

Comment:
promises are never fulfilled (or executed) if you dont have a `.then` (or equivalent) call. 
In this case its ok, since the Hello IDP is unused now.

Reply:
TIL

Added `.catch(console.error)` to log just errors
https://github.com/fastenhealth/fasten-onprem/pull/462/commits/e4570882a0b28c33899596a8a09980c035fb1e41